plugins/catalog/src/components/UnregisterEntityDialog/UnregisterEntityDialog.tsx

```diff
@@ -82,23 +99,38 @@ export const UnregisterEntityDialog = ({
       <DialogTitle id="responsive-dialog-title">
         Are you sure you want to unregister this entity?
       </DialogTitle>
+
       <DialogContent>
         {loading ? <Progress /> : null}
+
         {error ? (
           <Alert severity="error" style={{ wordBreak: 'break-word' }}>
-            {error.toString()}
+            {error.name === 'DeniedLocationException' ? (
+              <>
+                You cannot unregister this entity, since it originates from a
+                protected Backstage configuration (location
+                {`"${(error as DeniedLocationException).locationName}"`}). If
+                you believe this is in error, please contact the{' '}
+                {configApi.getOptionalString('app.title') ?? 'Backstage'}{' '}
+                operator.
```

Comment:
...total semantics, but wonder if we should align the two: https://backstage.io/docs/glossary
Perhaps adding operator to the glossary? I agree that sounds good in this context, though purely "integrator" would align to the glossary.

Reply:
Good point 👍

Reply:
Hm yeah, then again, for a (perhaps even non-technical) person that uses Backstage and sees the error they might be best helped by a formulation closer to "the team that manages Backstage" or something like that, of course ideally with a clickable link.

Reply:
There is this [`SupportButton`](https://github.com/backstage/backstage/blob/7abe2c2c3336878207c57094a739462981042f29/packages/core/src/components/SupportButton/SupportButton.tsx#L57-L61) that is used elsewhere, though not yet configured via the config but with props. But it being a button maybe not the best fit here?

![image](https://user-images.githubusercontent.com/720821/105154400-cef61880-5b09-11eb-9012-d7bc9093f70d.png)

Reply:
It's OK to leave this out for now, and yeah that specific button is meant to be used in a per-plugin page context.